lib/common/serverutil.c

```diff
@@ -25,7 +25,9 @@
 #include <pthread.h>
 #include <pwd.h>
 #include <signal.h>
+#ifndef __ANDROID__
```

Comment:
Could you please change this to `#ifndef __linux__`? That's when we use our own implementation instead of `posix_spawnp`.